src/pages/backmystack.js

```diff
@@ -0,0 +1,119 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+
+export default class BackMyStack extends React.Component {
+  static getInitialProps({ req, query }) {
+    const uuid = query.uuid; // handle case where the uuid is not supplied
+    const opencollectiveBaseUrl = process.env.OPENCOLLECTIVE_BASE_URL;
+    const openCollectiveRedirectUrl = `${opencollectiveBaseUrl}${process.env.OPENCOLLECTIVE_REDIRECT_PATH}`;
+
+    let protocol = 'https:';
+    const host = req ? req.headers.host : window.location.hostname;
```

Comment:
When I tested, the port was missing. Not sure if it was the case with or without `req`.

`window.location.host` perhaps?